src/core/webview/ClineProvider.ts

```diff
@@ -1043,6 +1044,7 @@ export class ClineProvider
 					<link href="${codiconsUri}" rel="stylesheet" />
 					<script nonce="${nonce}">
 						window.IMAGES_BASE_URI = "${imagesUri}"
+						window.ICONS_BASE_URI = "${iconsUri}"
```

Comment:
why?

Reply:
because the gutter animation is apparantly in icons

Reply:
I've had quite some trouble with some constants, because they werent shared between backend and guy, but you're implying I can just load it as a constant here? So all places doing this are silly?